fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -489,8 +489,16 @@ private void finalizeCommand() throws IOException {
                 // ShowResultSet is null means this is not ShowStmt, use remote packet(executor.getOutputPacket())
                 // or use local packet (getResultPacket())
                 if (resultSet == null) {
-                    packet = executor.getOutputPacket();
-                } else {
+                    if (executor.sendResultToChannel(ctx.getMysqlChannel())) {  // query statement result
+                        packet = getResultPacket();
+                        if (packet == null) {
+                            LOG.debug("packet == null");
```

Comment:
What's the case?

Reply:
Just removed these codes. Thanks for your reminder.